app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/CollectionServiceStarter.java

```diff
@@ -2,6 +2,9 @@
 
 import android.app.AlarmManager;
```

Comment:
Yes, probably:(  I'll remove it.  I guess this PR is not ready, obviously, to be merged.  In fact, I am still working on it and have found another issue in G5CollectionService re: Scan Constantly while testing.  So I'll pushing some changes to G5CollectionService as well that I hope you will review.